app-sso/getting-started/provision-auth-server.hbs.md

```diff
@@ -24,209 +25,75 @@ tanzu package installed list -A | grep "sso.apps.tanzu.vmware.com"
 For more information about the AppSSO installation,
 see [Install AppSSO](../how-to-guides/platform-operators/installation.md).
 
-## Provision an AuthServer
+## Discover Existing `AppSSO` Service Offerings
 
-Deploy your first Authorization Server along with an `RSAKey` key for signing tokens.
+AppSSO Login servers are a consumable service offering in TAP. The `ClusterWorkloadRegistrationClass` represents
+these service offerings.
+
+In your Kubernetes cluster, run the following command:
+
+```bash
+   tanzu services classes list
+```
+
+Assuming there isn't already a Login Offering you want to connect to, you will want to create your own. 
 
 > **Caution** This `AuthServer` example uses an unsafe testing-only identity provider. Never use it in
 production environments. For more information about identity providers, see [Identity providers](../how-to-guides/service-operators/identity-providers.md).
 
-```yaml
----
+## Set Up Your First `ClusterUnsafeTestLogin`
+
+In a non-poduction environment, [`ClusterUnsafeTestLogin`](../../reference/api/clusterunsafetestlogin.hbs.md) is the
+recommended way to get started with Application Single Sign-On (commonly called AppSSO). 
+
+```bash
+cat <<EOF | kubectl apply -f -
 apiVersion: sso.apps.tanzu.vmware.com/v1alpha1
-kind: AuthServer
-metadata:
-  name: my-authserver-example
-  namespace: default
-  labels:
-    name: my-first-auth-server
-    env: tutorial
-  annotations:
-    sso.apps.tanzu.vmware.com/allow-client-namespaces: "default"
-    sso.apps.tanzu.vmware.com/allow-unsafe-issuer-uri: ""
-    sso.apps.tanzu.vmware.com/allow-unsafe-identity-provider: ""
-spec:
-  replicas: 1
-  tls:
-    deactivated: true
-  identityProviders:
-    - name: "internal"
-      internalUnsafe:
-        users:
-          - username: "user"
-            password: "password"
-            email: "user@example.com"
-            emailVerified: true
-            roles:
-              - "user"
-  tokenSignature:
-    signAndVerifyKeyRef:
-      name: "authserver-signing-key"
-
----
-apiVersion: secretgen.k14s.io/v1alpha1
-kind: RSAKey
+kind: ClusterUnsafeTestLogin
 metadata:
-  name: authserver-signing-key
-  namespace: default
-spec:
-  secretTemplate:
-    type: Opaque
-    stringData:
-      key.pem: $(privateKey)
-      pub.pem: $(publicKey)
+  name: my-unsafe-test-login-example
```

Comment:
**[suggestion]:** You may know that I am on a crusade against [Hungarian-notation](https://en.wikipedia.org/wiki/Hungarian_notation) 😋 Therefore I would like to suggest something simpler. After all, the issuer URI will be `http://<name>-unsafe.<domain>`.
```suggestion
  name: my-login
```

Reply:
Done!